src/main/java/uk/gov/hmcts/darts/ws/ContextRegistryEndpoint.java

```diff
@@ -34,11 +34,12 @@ public JAXBElement<RegisterResponse> register(@RequestPayload JAXBElement<docume
         RegisterResponse registerResponse = new RegisterResponse();
 
         try {
+
             // create a session as the client needs this
-            Optional<Token> token = registerable.store(registerable.createValue(addDocument.getValue().getContext()));
+            Optional<Token> cacheValue = registerable.store(addDocument.getValue().getContext());
```

Comment:
Should the `addDocument` param be renamed here to be `register` here?
That would seem in keep with the convention below in `unregister`